app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
[OPTIONAL] Excellent work implementing mostly features from `devise`, as this is going to be an application that you will show in your Job Searching, it would be great to completely implement the `:confirmable`, so the user can receive the email and confirm it. Or just remove it for a better UX. Just my suggestion, great job! 🥳

Reply:
@BregornOriginal thanks for the feedback ,  we have implement  for `:confirmable`